common/scala/src/main/scala/whisk/core/containerpool/Container.scala

```diff
@@ -142,12 +142,16 @@ trait Container {
     callContainer("/run", body, timeout, retry = false)
       .andThen { // never fails
         case Success(r: RunResult) =>
-          transid.finished(
-            this,
-            start.copy(start = r.interval.start),
-            s"running result: ${r.toBriefString}",
-            endTime = r.interval.end,
-            logLevel = InfoLevel)
+          r.response match {
+            case Left(_) => transid.failed(this, start, s"run failed with ${r.toBriefString}", logLevel = ErrorLevel)
```

Comment:
Can this be caused by user behavior or is this guaranteed to be an error of the system?

Reply:
I think so as the `ContainerConnectionError` here only occurred while connecting to action containers, or is there any thing users can do to terminate/prevent such connections?

Reply:
For blackbox actions for example, the run path can be broken.

Reply:
ok, but that should be marked as error too in my opinion , as the activation is failed anyway

Reply:
It is not an ERROR in the sense of the system is at fault though. Ideally, you'd want to monitor "ERROR" log messages in your production system and judge based on that if something is wrong. That loglevel should never be in the hand of the user of the system imo.